ci-script-windows.sh

```diff
@@ -56,13 +56,8 @@ execute(){
 build_darktable() {
     cd "$(dirname "$0")"
 
-    # for RawSpeed submodule
-    git submodule init
-    git submodule update
-
-    mkdir artifacts
     mkdir build && cd build
-    cmake -G "MSYS Makefiles" -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=/c/projects/darktable/artifacts ../.
+    cmake -G "MSYS Makefiles" -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=/c/projects/darktable/build ../.
```

Comment:
So actually @LebedevRI is right. In order to make `ninja` work custom shell script command needs a `bash `before the script name, plus the few perl script we have (like `intltool-merge`) needs a `perl` command before the script name. I guess these changes are not intrusive and would not do any harm on Linux build either.
So with a minimal modifications I was able to use the ninja build system as well to compile dt.
Would you like me to add those changes to this PR?